include/drivers/sensor.h

```diff
@@ -90,6 +90,8 @@ enum sensor_channel {
 	SENSOR_CHAN_HUMIDITY,
 	/** Illuminance in visible spectrum, in lux. */
 	SENSOR_CHAN_LIGHT,
+	/** Ultraviolet index, in UVI. */
+	SENSOR_CHAN_UV,
```

Comment:
Same comment from the previous PR:
> I wouldn't put this in the middle of the other light channels given that the units are not the same.
Its also not obvious that SENSOR_CHAN_UV and SENSOR_CHAN_IV are returning very different things.
Perhaps SENSOR_CHAN_UV_INDEX?

Reply:
UV is always measured in the UVI scale(see https://en.wikipedia.org/wiki/Ultraviolet_index).

Yes, the placement of the channel is not quite right.

Reply:
I'm not sure that's strictly accurate, sure that's the normal day-to-day usage, but I would think irradiance would be the normal scientific term. The current definition for `SENSOR_CHAN_IR` is obviously wrong though, as lux only has meaning in visible wavelengths. A more consistent solution would be:
```
SENSOR_CHAN_IR, /* Irradiance, W/m^2 */
SENSOR_CHAN_UV, /* Irradiance, W/m^2 */
SENSOR_CHAN_UV_INDEX /* Ultraviolet Index, unitless */
```